### ISSUE TITLE
run exported html fails intermittently

### ISSUE THREAD

**Issue by samtsejerome** [godotengine/godot#50585]
### Godot version

3.3.2 stable mono

### System information

MacOs 10.15.7 on macbook pro

### Issue description

clicking the  'run exported html' button in the editor will sometimes run my project in the browser and other  times fail this makes a consistent development workflow impossible. and I haven't been able to find information about any logs documented anywhere to try and find any detail (the output message is v general)
see screenshot attached:
<img width="1120" alt="issue-sshot" src="https://user-images.githubusercontent.com/68197715/126075181-ee5e7c81-00e0-493d-a25b-d4fd087da739.png">

### Steps to reproduce

as I explained in the summary it intermittently occurs so I cant  document consistently reliable steps to reproduce, (particularly as there is no error detail to gain any insight into the cause)

### Minimal reproduction project

this only occurs in my main (significant sized) project, perhaps you could share some information about logged detail and I may be able to help more..

**Comment by Calinou** [comment]
Try starting the Godot editor with the `--verbose` [command line argument](https://docs.godotengine.org/en/stable/getting_started/editor/command_line_tutorial.html).

**Comment by samtsejerome** [comment]
i've never run the editor from terminal  before - does it work in macos ?

**Comment by Calinou** [comment]
> i've never run the editor from terminal before - does it work in macos ?

Yes, by using `cd` to reach the .app folder then running `Contents/MacOS/Godot`.

**Comment by samtsejerome** [comment]
ok - thanks.. i tried that with the --verbose arg and it worked, though as
you'd expect this time it ran without error!? though next time it happens i
can look for some output now - thx!

**Comment by samtsejerome** [comment]
although when I try again it doesn't work ( I dont know of anything I did differently!?)

![image](https://user-images.githubusercontent.com/68197715/126910448-ee5580b9-ce1e-442f-92f1-6a20832101f2.png)

**Comment by samtsejerome** [comment]
figured it out - sorry for the noise...

thx v much, log output will be handy for more than this issue..  :)
